[DOCS] What's new in 8.3 (#2155)
elastic/security-docs: docs/whats-new.asciidoc

[role="screenshot"]
image::whats-new/images/8.3/grouped-nav.png[]

[discrete]
== New dashboards provide a summary of critical information

Two new dashboards have been added to the {securty-app} to provide a visualization of critical information.

The {securty-guide}/overview-dashboard.html[*Overview* dashboard] provides a high-level snapshot of detections, external alerts, and event trends. Use this dashboard to assess overall system health and find anomalies that may require further investigation.

[role="screenshot"]
image::whats-new/images/8.3/overview-pg.png[]

The {securty-guide}/detection-response-dashboard.html[*Detections & Response* dashboard] dashboard provides focused visibility into the daily operations of your security environment. Use this dashboard to monitor recent and high priority detection alerts and cases, and identify the top hosts and users associated with the most alerts so you can triage effectively.

[role="screenshot"]

> image::whats-new/images/8.3detection-response-dashboard.png[]

Author: @benironside yes -- once I checked the logs this was the problem and threw everything else off. Thanks ;)

[discrete]
== Technical preview features

*Introducing Cloud Security Posture Management*

{security-guide}/security-posture-management.html[*Cloud Security Posture Management*] helps you compare your cloud and Kubernetes infrastructure to security best practices.

*User risk score*

https://github.com/elastic/detection-rules/blob/209b40b0a30d87898d75bb2d5dc3f2e068b5f09d/docs/experimental-machine-learning/user-risk-score.md[User risk score] assigns a score to highlight risky users from within your environment. It utilizes a transform with a scripted metric aggregation to calculate scores based on detection rule alerts with an "open" status within a 5-day window. The transform runs hourly to update the score as new detection rule alerts are generated. Each user risk score is normalized to a scale of 0 to 100.

[discrete]
== New Authentications tab added to Users page

An *Authentications* tab has been added to the Users page to show how many hosts a user has authenticated.

[role="screenshot"]
image::whats-new/images/8.3/user-auth.png[]

[discrete]
== Detection rules enhancements

*New optional settings for event correlation rules*

> {securty-guide}/rules-ui-create.html#create-eql-rule[Event correlation rules] have new optional settings for specifying EQL fields: *Event category*, *Tiebreaker*, and *Timestamp* fields.

Reviewer: ```suggestion
{security-guide}/rules-ui-create.html#create-eql-rule[Event correlation rules] now allow you to specify the EQL fields *Event category*, *Tiebreaker*, and *Timestamp*.
```

*{ml} rules upgraded to V3 {ml} jobs*

> Elastic prebuilt machine learning detection rules for some Windows and Linux anomalies have been updated with new V3 {ml}} jobs. Refer to our {securty-guide}/alerts-ui-monitor.html#ml-job-compatibility[documentation] for information about how to  upgrade and/or continue to use the old V1/V2 jobs.

Reviewer: ```suggestion
Elastic prebuilt detection rules for some Windows and Linux anomalies have been updated with new V3 {ml} jobs. Refer to our {security-guide}/alerts-ui-monitor.html#ml-job-compatibility[documentation] for information about how to upgrade and/or continue to use the old V1/V2 jobs.
```

*New Actions column in rule execution logs table enables filtering*

You can create a {security-guide}/alerts-ui-monitor.html#rule-execution-logs[global search filter] based on a specific rule execution by selecting the filter icon in the *Actions* column of the *Rule execution logs* tab on the rule details page. Enabling this filter replaces any previously applied filters.

[role="screenshot"]
image::whats-new/images/8.3/actions-icon.png[]

*New prebuilt rules*

> 15 new {security-guide}/prebuilt-rules.html[prebuilt rules] have been added in this release.

Reviewer: ```suggestion
15 new {security-guide}/prebuilt-rules.html[prebuilt rules] were added.
```

[discrete]
== OAuth support in ServiceNow connectors

> The ServiceNow connectors now support open authentication (OAuth).

Reviewer 2: ```suggestion
The {sn} connectors now support open authentication (OAuth).
```

For configuration details, refer to
{kibana-ref}/servicenow-action-type.html[ServiceNow ITSM], {kibana-ref}/servicenow-sir-action-type.html[ServiceNow SecOps],
and {kibana-ref}/servicenow-itom-action-type.html[ServiceNow ITOM connector].

[discrete]
== Cases enhancements

The following enhancements have been added to Cases:

> * You can now can assign a severity level to cases.

Reviewer: ```suggestion
* You can now can assign severity levels to cases.
```

+

> NOTE: If you do not set the case's severity, it defaults to Low.

Reviewer: ```suggestion
NOTE: If you do not set a case's severity, it defaults to Low.
```

+

> * The Cases table now includes a *Severity* column and an option to filter the table by severity. The average time to close case metrics has also been added to the Cases table.

Reviewer: ```suggestion
* The Cases table now includes a *Severity* column and an option to filter by severity. It also now includes an "Average time to close" metric.
```

* You can now delete text comments, including those in Lens visualizations.

> * You can add multiple alerts to new and existing cases through the *Bulk actions* menu.

Reviewer: ```suggestion
* You can now add multiple alerts to new and existing cases via the *Bulk actions* menu.
```

Author: @benironside per UX guidelines, we are doing away with `via`.

> * A new *Alerts* tab has been added to the case details page. This allows users to view all alerts attached to a case.

Reviewer: ```suggestion
* A new *Alerts* tab has been added to the case details page. This allows you to view all alerts attached to a case.
```

[discrete]
== Alert details enhancements

The following enhancements have been added to the alert details flyout:

> * Each count value in the Alert prevalence column is now an active link that when selected, allows you to explore that field value in Timeline.

Reviewer: ```suggestion
* Numerical values in the Alert prevalence column are now active links that send you to a Timeline to investigate the selected alert(s).
```

+
[role="screenshot"]
image::whats-new/images/8.3/alert-prevalance.gif[]

> * The Session ID, which provides a unique ID for tracking a Linux session, has been added to the *Highlighted fields* section. To collection session view data, you must {security-guide}/session-view.html#enable-session-view[enable session view data] in an integration policy.

Reviewer: ```suggestion
* Session ID, a unique ID for Linux sessions, has been added to the *Highlighted fields* section. To collect session data from Linux hosts, you must {security-guide}/session-view.html#enable-session-view[enable session data] in an integration policy.
```

[discrete]
== Osquery enhancements

> For more Osquery enhancements, check out the {kibana-ref}/whats-new.html#highlights-8.3-osquery[{kib} Osquery highlights].

Reviewer: ```suggestion
For information about additional Osquery enhancements, check out the {kibana-ref}/whats-new.html#highlights-8.3-osquery[{kib} Osquery highlights].
```
Maybe this should go after line 141?

{security-guide-all}/7.9/whats-new.html[7.9]

// NOTE: The notable-highlights tagged regions are re-used in the Installation and Upgrade Guide. Full URL links are required in tagged regions.
// tag::notable-highlights[]

[discrete]
[[term-changes-8.3]]
== Terminology changes

*"Endpoint Security integration" has been renamed to "Endpoint and Cloud Security integration"*

Due to the launch of https://www.elastic.co/security/cloud-security[Elastic Security for Cloud], the *Endpoint Security integration*, which allows the {agent} to monitor for events on your host, has been renamed to *Endpoint and Cloud Security integration*. Please note that general industry term references to endpoint security have not changed.

[role="screenshot"]
image::whats-new/images/8.3/cloud-integration.png[]

[discrete]
[[features-8.3]]

[discrete]
== New streamlined navigation

An optional, new navigation menu, which can be enabled in the {security-guide}/advanced-settings.html#_enable_grouped_navigation[advanced {kib} settings], groups related pages and highlights commonly visited areas for a streamlined experience.

[role="screenshot"]

> image::whats-new/images/8.3/grouped-nav.png[Grouped navigation menu]

Reviewer 3: Image quality for this screenshot looks low-res -- could we use a direct copy of the version at `docs/getting-started/images/grouped-nav-ui.png`? It looks like the file might have gotten saved from high-res (Retina) to lower-res when it was copied into the whats-new folder.

Or just use the original image reference from the docs -- I checked this locally and it should resolve to the same image, same dimensions:
```suggestion
image::getting-started/images/grouped-nav-ui.png[width=75%][height=75%][Grouped navigation menu]
```

[role="screenshot"]
image::whats-new/images/8.3/overview-pg.png[]

The {security-guide}/detection-response-dashboard.html[*Detection & Response* dashboard] provides focused visibility into the daily operations of your security environment. Use it to monitor recent and high priority detection alerts and cases, and identify the top hosts and users associated with the most alerts so you can triage effectively.

[role="screenshot"]
image::whats-new/images/8.3/detection-response-dashboard.png[]

[discrete]
== New integrations

Several new https://docs.elastic.co/integrations[integrations] have been added, including ones for CIS Kubernetes Benchmark, AWS Security Hub, Cloudflare, Jamf, and Palo Alto Networks.

[discrete]
== Technical preview features

*Cloud Security Posture Management*

Cloud Security Posture Management (CSPM) and Kubernetes security posture management (KSPM) help you compare your cloud and Kubernetes infrastructure to security best practices. Refer to our {security-guide}/security-posture-management.html[documentation] for setup instructions.

*User risk score*

> https://github.com/elastic/detection-rules/blob/209b40b0a30d87898d75bb2d5dc3f2e068b5f09d/docs/experimental-machine-learning/user-risk-score.md[User risk score] assigns a score to highlight risky users within your environment. It utilizes a transform with a scripted metric aggregation to calculate scores based on detection rule alerts within a 90-day window. The transform runs hourly to update the score as new detection rule alerts are generated. Each user risk score is normalized on a scale of 0 to 100.

Reviewer 3: ```suggestion
https://github.com/elastic/detection-rules/blob/209b40b0a30d87898d75bb2d5dc3f2e068b5f09d/docs/experimental-machine-learning/user-risk-score.md[User risk score] assigns a score to highlight risky users within your environment. It uses a transform with a scripted metric aggregation to calculate scores based on detection rule alerts within a 90-day window. The transform runs hourly to update the score as new detection rule alerts are generated. Each user risk score is normalized on a scale of 0 to 100.
```

[discrete]
== New Authentications tab added to Users page

An *Authentications* tab has been added to the Users page to show successful and failed authentication events per user.

[role="screenshot"]
image::whats-new/images/8.3/user-auth.png[]

[discrete]
== Detection rules enhancements

*New optional settings for event correlation rules*

{security-guide}/rules-ui-create.html#create-eql-rule[Event correlation rules] now allow you to specify the following EQL fields: *Event category*, *Tiebreaker*, and *Timestamp* fields.

> *{ml-cap} rules upgraded to V3 {ml} jobs*

Reviewer 3: I started using lowercase `v3` since the ML job names all use lowercase. Not sure if we use code formatting in sub-headings like this:

```suggestion
*{ml-cap} rules upgraded to `v3` {ml} jobs*
```

> Elastic prebuilt rules for some Windows and Linux anomalies have been updated with new V3 {ml}} jobs. Refer to our {security-guide}/alerts-ui-monitor.html#ml-job-compatibility[documentation] for information about how to upgrade and/or continue to use the old V1/V2 jobs.

Reviewer 3: ```suggestion
Elastic prebuilt rules for some Windows and Linux anomalies have been updated with new `v3` {ml} jobs. Refer to our {security-guide}/alerts-ui-monitor.html#ml-job-compatibility[documentation] for information about how to upgrade and/or continue to use the old `v1`/`v2` jobs.
```

+
* The Cases table now includes a *Severity* column and an option to filter the table by severity. It also now includes an "Average time to close" metric.
* You can now delete text comments, including those in Lens visualizations.
* You can now add multiple alerts to new and existing cases through the *Bulk actions* menu.
* A new *Alerts* tab has been added to the case details page. This allows you to view all alerts attached to a case.

[discrete]
== Alert details enhancements

The following enhancements have been added to the alert details flyout:

* Numerical values in the Alert prevalence column are now active links that send you to Timeline, where you can investigate related alerts.
+
[role="screenshot"]
image::whats-new/images/8.3/alert-prevalance.gif[]
* Session ID, a unique ID for Linux sessions, has been added to the *Highlighted fields* section. To collect session data from Linux hosts, you must {security-guide}/session-view.html#enable-session-view[enable session view data] in an integration policy.

[discrete]
== Osquery enhancements

You can now run Osquery from the *More actions* menu in the Alerts table.

[role="screenshot"]

> image::whats-new/images/8.3/run-osquery.png[]

Reviewer 3: Nit: Border on this image is a little inconsistent (no border on left side).
